packages/@ionic/cli-utils/src/lib/project/angular/ailments.ts

```diff
@@ -8,11 +8,77 @@ import { pkgFromRegistry, pkgManagerArgs } from '../../utils/npm';
 import { Project as AngularProject } from './';
 
 export function registerAilments(registry: IAilmentRegistry, deps: AutomaticallyTreatableAngularAilmentDeps) {
-  registry.register(new IonicForAngularUpdateAvailable(deps));
-  registry.register(new IonicForAngularMajorUpdateAvailable(deps));
-
-  // TODO: @ionic/core update available
-  // TODO: Angular CLI update available
+  // for @ionic/angular
+  registry.register(new UpdateAvailable(deps, {
+    id: 'ionic-for-angular-update-available',
+    pkgName: '@ionic/angular',
+    treatmentVisitURL: ['https://github.com/ionic-team/ionic/releases'],
+  }));
+  registry.register(new MajorUpdateAvailable(deps, {
+    id: 'ionic-for-angular-major-update-available',
+    pkgName: '@ionic/angular',
+    treatmentVisitURL: ['https://blog.ionicframework.com', 'https://github.com/ionic-team/ionic/releases'],
+  }));
+
+  // @ionic/core
```

Comment:
`@ionic/core` won't be in the list of dependencies for starters in the stable release, so this check isn't needed